### PR TITLE
Output all known errors in all forms.

### PR DESCRIPTION
**Pull request by henrahmagix**
>
Known errors:
- `detail` rate throttling.
- `non_field_errors` any error on the form that
  isn't field specific.
>
Any other errors keys from django-user-management
will need to be output also.

If anyone knows of any other error keys coming from django-user-management, please post here

@incuna/frontend Please merge, ta!

### CODE REVIEW

File: src/password/templates/user_management/password/reset_request_form.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <label for="email" ng-if="fields.email.label">{{ fields.email.label }}</label>
     <input type="email" ng-model="data.email" class="form-control" id="email" autofocus>
     <div class="help-block" ng-if="errors.email" ng-bind="errors.email"></div>
+    <div class="help-block" ng-if="errors.detail" ng-bind="errors.detail"></div>
+    <div class="help-block" ng-if="errors.non_field_errors" ng-bind="errors.non_field_errors"></div>
 </div>
 
+
```
Comment (wytrych) [inline src/password/templates/user_management/password/reset_request_form.html]: Redundant newline?

Comment (henrahmagix) [inline src/password/templates/user_management/password/reset_request_form.html]: Good spot!

Comment (henrahmagix) [inline src/password/templates/user_management/password/reset_request_form.html]: Updated with force push

File: src/sms-verification/templates/user_management/sms-verification/verify_form.html [hunk 2]
```diff
@@ -16,9 +16,14 @@
     <div class="help-block" ng-if="errors.activation_code" ng-bind="errors.activation_code"></div>
 </div>
 
+<!-- This module does not parse an array of errors into a single error string
+    so we must loop over the property on scope.errors. -->
 <div ng-show="errors.non_field_errors" ng-class="{'has-error': errors.non_field_errors}">
     <div ng-repeat="error in errors.non_field_errors" class="help-block" ng-bind="error"></div>
```
Comment (wytrych) [inline src/sms-verification/templates/user_management/sms-verification/verify_form.html]: So why do we need to `ng-repeat` only in this template?

Comment (henrahmagix) [inline src/sms-verification/templates/user_management/sms-verification/verify_form.html]: Array: https://github.com/incuna/angular-user-management/blob/ca68ae97af13cb30fb3b58644075bbf38aba085f/src/sms-verification/scripts/directives.js#L34

vs

Single: https://github.com/incuna/angular-user-management/blob/ca68ae97af13cb30fb3b58644075bbf38aba085f/src/sms-password/scripts/directives.js#L35-L43

The former is the only instance in this library: it should be updated to be like the latter.

Comment (wytrych) [inline src/sms-verification/templates/user_management/sms-verification/verify_form.html]: :+1:

**Comment by henrahmagix** [comment]
Thanks!

**Pull request merged**